cEP-0029.md

```diff
@@ -0,0 +1,63 @@
+
+## Changes to IGitt
+
+Docstrings, wherever missing, need to be added in IGitt codebase ([#146](https://gitlab.com/gitmate/open-source/IGitt/-/issues/146)). Also, IGitt documentation needs to be updated to reflect changes occurring due to migration from Gitmate to coala organisation. We can also add a section on how IGitt codebase is structured, include examples for those actions missing currently like notifications, reactions etc, and add a section on how to use and set up and use for developers. Cron jobs are currently missing for different versions of python, so they need to be added after adding support for latest versions. Lastly, coala-IGitt needs to be published as a seperate package.
+
+## Changes to Gitmate
+
+### Fix CI/CD and make Gitmate functional
+
+Firstly, cron jobs need to be added for all the supported versions of python. Also, Gitmate hosting needs to be migrated to a new domain before we activate Gitmate for the coala community.
+
+### Improve frontend
+
+Multiple improvements can be made to the frontend. These include **UI changes** such as:
+
+* Removing the use of German
+* Removing the list of sponsor companies
+* Revamp landing page
+* Copy plugin settings shows ‘suggested password’
+* Make spinner go away after activating gitmate on a repository
+* Slide toggles buggy on several occasions - replace with a better alternative
+* [#104](https://gitlab.com/gitmate/open-source/gitmate-2-frontend/-/issues/104) - Fix alignment of toggles for suboptions
+* Allow user to go to ‘/repo/n’ by click on repository card
+* [#83](https://gitlab.com/gitmate/open-source/gitmate-2-frontend/-/issues/83) - Add repository name in title
+* [#81](https://gitlab.com/gitmate/open-source/gitmate-2-frontend/-/issues/81) - Use material chips in text areas
+* [#66](https://gitlab.com/gitmate/open-source/gitmate-2-frontend/-/issues/66) - Make ‘copy plugin’ bar longer
+* Improve design of repository cards
+
+**Frontend documentation** can be improved and the following changes can be made:
+
+* Improve developers section - Fix broken links, improve part about setting environment variables
+* Complete FAQ.md
+* Add a section on how the Gitmate codebase is structured.
+
+### Changes to backend
+
+We can **add docstrings** to the backend codebase wherever missing. According to [#242](https://gitlab.com/gitmate/open-source/gitmate-2/-/issues/242), we need to **merge the ‘ack’, ‘approver’, and ‘auto label pending or wip’ plugins** into a combined 'review' plugin. This will also solve the following issues - [#144](https://gitlab.com/gitmate/open-source/gitmate-2/-/issues/144), [#243](https://gitlab.com/gitmate/open-source/gitmate-2/-/issues/243), [#375](https://gitlab.com/gitmate/open-source/gitmate-2/-/issues/375), [#376](https://gitlab.com/gitmate/open-source/gitmate-2/-/issues/376)
```

Comment:
List please.

Reply:
Also add code changes wherever required, this improves the readability of the cEP.